webapp/channels/src/packages/mattermost-redux/src/selectors/entities/scheduled_posts.ts

```diff
@@ -56,13 +56,18 @@ export function hasScheduledPostError(state: GlobalState, teamId: string) {
 }
 
 export function showChannelOrThreadScheduledPostIndicator(state: GlobalState, channelOrThreadId: string): ChannelScheduledPostIndicatorData {
-    const channelScheduledPosts = state.entities.scheduledPosts.byChannelOrThreadId[channelOrThreadId] || emptyList;
+    const allChannelScheduledPosts = state.entities.scheduledPosts.byChannelOrThreadId[channelOrThreadId];
+    const eligibleScheduledPosts = allChannelScheduledPosts.filter((scheduledPostId: string) => {
+        const scheduledPost = state.entities.scheduledPosts.byId[scheduledPostId];
+        return !scheduledPost.error_code;
```

Comment:
is there a type for the scheduledPost object? wondering why the snake case for error_code?

Reply:
So all the types that for API responses have camel case, like this one